### DISCUSSION TITLE
Detecting “Harvest Now, Decrypt Later” Attacks Using AI/ML Models

### DISCUSSION THREAD

**Discussion by vkosuri** [open-quantum-safe/liboqs#1835] [General]
I’m researching strategies to detect the “[Harvest Now, Decrypt Later](https://en.wikipedia.org/wiki/Harvest_now,_decrypt_later)” attack

Specifically, I’m interested in exploring AI and machine learning models that can identify signs of this attack.

Are there any existing models or techniques that can help detect instances of data harvesting with the intention of future decryption?

**Comment by dstebila** [comment]
Companies offering network monitoring or network intrusion detection software may have heuristics aiming to detect this. But it's outside the scope of what we do in the OQS project, which is about implementation of post-quantum algorithms.
